src/pages/BrowseSchemes.svelte

```diff
@@ -202,21 +206,17 @@
           <CollapsibleCard
             label={`${scheme.scheme_reference}: ${scheme.num_features} features`}
           >
-            <ul>
-              <li>Authority or region: {scheme.authority_or_region}</li>
-              <li>Capital scheme ID: {scheme.capital_scheme_id}</li>
-              <li>Funding programme: {scheme.funding_programme}</li>
-              <li>
-                <button type="button" on:click={() => showScheme(scheme)}
-                  >Show on map</button
-                >
-              </li>
-              <li>
-                <button type="button" on:click={() => editScheme(scheme)}
-                  >Edit scheme</button
-                >
-              </li>
-            </ul>
+            <p>Authority or region: {scheme.authority_or_region}</p>
+            <p>Capital scheme ID: {scheme.capital_scheme_id}</p>
+            <p>Funding programme: {scheme.funding_programme}</p>
+            <div class="govuk-button-group">
```

Comment:
Made two of the buttons side-by-side. I'm sure the contents of these expandable cards will keep evolving, so not changing much else yet